### PR TITLE
user/:id page

### PR DESCRIPTION
**Pull request by Damrod**
we need a ui to see user pages other than for self

### CODE REVIEW

File: webapp/src/index.tsx [hunk 1]
```diff
@@ -24,7 +24,8 @@ root.render(
         <Route path={USERS_URL} element={<Users />} />
         <Route path={PLAY_URL} element={<Play />} />
         <Route path={CHAT_URL} element={<Chat />} />
-        <Route path={USER_URL} element={<UserPage />} />
+        <Route path={USER_URL} element={<UserPage isMe={true} />} />
+        <Route path={`${USERS_URL}/:id`} element={<UserPage isMe={false} />} />
```
Comment (gbudau) [inline webapp/src/index.tsx:28]: Very cool :rocket:

**Review by gbudau — APPROVED** [review]
I like how you solve this by adding a prop to the `UserPage`!
LGTM :tada:

File: webapp/src/index.tsx (hunk 1 above)
Comment (Damrod) [reply webapp/src/index.tsx:28]: thx!

Comment (Damrod) [reply webapp/src/index.tsx:28]: thx!

**Review by fizcamposancos — COMMENTED** [review]
Good job!!

**Pull request merged**